src/main/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/Implementation/CoffeeMaker.java

```diff
@@ -27,6 +27,75 @@ public CoffeeMaker() {
         deliveryPipe = new DeliveryPipe();
     }
 
+    /**
+     * Fill the boiler with cups of water
+     */
+    public void fillBoilerWithWater(final int cupsWater) {
+        boiler.setCupsOfWater(cupsWater);
+    }
+
+    /**
+     * Fill the boiler with cups of water
+     */
+    public void fillFilterWithCoffeeGrains() {
+        filterAndReceptacle.putCoffeeGrains();
+    }
+
+    /**
+     * Returns true if the pot is it's place
+     * @return boolean
+     */
+    public boolean obtainsStartButtonState() {
+        return startButton.getButtonPressed();
+    }
+
+    /**
+     * Returns true if the pot is it's place
```

Comment:
This documentation seems to be wrong

Reply:
Thanks for the observation, I already corrected it.